src/pasta/fields/fp.rs

```diff
@@ -164,69 +164,61 @@ impl_binops_additive!(Fp, Fp);
 impl_binops_multiplicative!(Fp, Fp);
 
 /// INV = -(p^{-1} mod 2^64) mod 2^64
-const INV: u64 = 0xa14064e1ffffffff;
+const INV: u64 = 0x992d30ecffffffff;
```

Comment:
I did not check this constant, but I'm satisfied by its unit test.

Reply:
I checked this in Sage.